src/Sensors.hpp

```diff
@@ -17,9 +17,10 @@
 #include <s8_uart.h>
 #include <sps30.h>
 #include <drivers/pm1006.h>
+#include <drivers/DFRobot_MultiGasSensor.h>
 
-#define CSL_VERSION "0.5.9"
-#define CSL_REVISION 366
+#define CSL_VERSION "0.5.9.1"
```

Comment:
this is invalid, only add one more to the final number of revision and version.

More info here:
https://docs.platformio.org/en/latest/manifests/library-json/fields/version.html

Reply:
Entendido

Reply:
Bueno, aqui deberia quedar 0.5.10 si fue un cambio pequeño compatible. O 0.6.0 si no es compatible hacia atras. Q creo es el caso por el tema del gas, porque ese getter depronto lo vamos a cambiar